packages/linter/src/cli.ts

```diff
@@ -0,0 +1,238 @@
+import { readFileSync } from "fs";
+import { isArray } from "util";
+
+import program from "commander";
+import fetch from "node-fetch";
+import cheerio from "cheerio";
+
+import { lint, Result, LintMode, guessMode } from ".";
+import { fetchToCurl } from "./helper";
+
+// import { version } from "../package.json";
+
+const UA = {
+  googlebot_mobile: [
+    "Mozilla/5.0 (Linux; Android 6.0.1; Nexus 5X Build/MMB29P) AppleWebKit/537.36",
+    "(KHTML, like Gecko) Chrome/41.0.2272.96 Mobile Safari/537.36",
+    "(compatible; Googlebot/2.1; +http://www.google.com/bot.html)"
+  ].join(" "),
+  googlebot_desktop: [
+    "Mozilla/5.0 AppleWebKit/537.36 (KHTML, like Gecko; compatible;",
+    "Googlebot/2.1; +http://www.google.com/bot.html) Safari/537.36"
+  ].join(" "),
+  chrome_mobile: [
+    "Mozilla/5.0 (Linux; Android 8.0; Pixel 2 Build/OPD3.170816.012)",
+    "AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.86 Mobile Safari/537.36"
+  ].join(" "),
+  chrome_desktop: [
+    "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3)",
+    "AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.86 Safari/537.36"
+  ].join(" ")
+};
+
+export function cli(argv: string[]) {
+  program
+    // .version(version)
+    .usage(`amplint [options] URL|copy_as_cURL`)
+    .option(
+      `-f, --force <string>`,
+      "override test type",
+      /^(auto|sxg|amp|ampstory)$/i, // needs to be of type LintMode | "auto"
+      "auto"
+    )
+    .option(
+      `-t, --format <string>`,
+      "override output format",
+      /^(text|json|tsv|html)$/i,
+      "text"
+    )
+    .option(
+      `-A, --user-agent <string>`,
+      "user agent string",
+      /^(googlebot_desktop|googlebot_mobile|chrome_desktop|chrome_mobile)$/i,
+      "googlebot_mobile"
+    )
+    .on("--help", function() {
+      console.log("");
+      console.log("Examples:");
+      console.log("  $ amplint https://www.ampproject.org/");
```

Comment:
replace with amp.dev ;-)

Reply:
Ha! Fixed in a5b25d71af56bff61f1e4f9ee4e5052603ac0b11.